### PR TITLE
Minizip

### PR DESCRIPTION
**Pull request by uilianries**
Specify library name and version:  **minizip/1.2.11**

I decided create this PR because:
- https://github.com/conan-io/conan-center-index/pull/386 requires minizip
- Both Hunter and VCPKG offer minizip as separated package from zlib

Why do we need a CMake file to create minizip?
The original zlib cmake does not offer a separated recipe only for minizip, also, this cmake file was borrowed from VCPKG.

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/minizip/1.2.11/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,80 @@
+cmake_minimum_required(VERSION 3.8)
```
Comment (danimtb) [inline recipes/minizip/1.2.11/CMakeLists.txt]: Was this Cmake file extracted from other place?

**Comment by danimtb** [comment]
So `mizip` is not included in the current Conan's zlib package then?

File: recipes/minizip/1.2.11/CMakeLists.txt (hunk 1 above)
Comment (uilianries) [reply recipes/minizip/1.2.11/CMakeLists.txt]: yes, from vcpkg: https://github.com/microsoft/vcpkg/blob/master/ports/minizip/CMakeLists.txt

**Comment by uilianries** [comment]
> So `minizip` is not included in the current Conan's zlib package then?

@danimtb By default, zlib package is not generating minizip. But we have minizip as optional there in zlib.

The thing is, if you want only minizip, we can use this library. Minizip requires zlib, but you don't care about, you are interested in minizip. The project assimp works on this way, take a look on vcpkg's assimp port.

**Comment by danimtb** [comment]
Thanks for the clarification. It makes sense

File: recipes/minizip/1.2.11/CMakeLists.txt (hunk 1 above)
Comment (danimtb) [reply recipes/minizip/1.2.11/CMakeLists.txt]: Can you please add that link as a comment in the CMakeLists.txt content? Something like "Extracted from"

Comment (uilianries) [reply recipes/minizip/1.2.11/CMakeLists.txt]: good idea!

Comment (uilianries) [reply recipes/minizip/1.2.11/CMakeLists.txt]: done

**Pull request merged**